profiler/src/main/java/com/splunk/opentelemetry/profiler/JfrPathHandler.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.profiler;
+
+import java.nio.file.Path;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+import jdk.jfr.consumer.RecordedEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Responsible for processing a single jfr file snapshot. It streams events from the
+ * RecordedEventStream into the processing chain and, once complete, calls the onComplete callback.
```

Comment:
Shouldn't it be `onFileFinished` instead?

```suggestion
 * RecordedEventStream into the processing chain and, once complete, calls the onFileFinished callback.
```

Reply:
Yup, I think a refactor didn't propagate into the comments.  Good catch.